src/System.Management.Automation/engine/TypeTable_GetEvent_Types_Ps1Xml.cs

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Management.Automation.Language;
+using System.Reflection;
+
+namespace System.Management.Automation.Runspaces
+{
+    public sealed partial class TypeTable
+    {
+        private void Process_GetEvent_Types_Ps1Xml(string filePath, ConcurrentBag<string> errors)
```

Comment:
Type tables could be updated whenever you import a module, so it's changed frequently. I don't see the benefit of making this static, it depends on the member table, type converter table, and the type adapter table of the specific type table instance.

Reply:
Clear about static.

> Type tables could be updated whenever you import a module, so it's changed frequently

How many modules updates a type table (specially in startup scenario)?

Reply:
Thanks!

Previously I did not found  on GitHub using the binary API for type table update but I found ~1000 usages of `Update-TypeData` cmdlet.
Also on my Windows Server 2019 with Admin tools installed I found ~86 files *.Types.ps1xml.
I think the Windows standard modules could benefit from the way used in the PR.
Although I'm not sure since these files are small.

These both suggest that only a few new types are added, but it forces us to make copies of the type table, which seems wasteful at first glance.